Fix HandleGMTicketUpdateTextOpcode

Elmsroth opened the pull request:
When updating the ticket , the client adds a leading '\a' char ! so we need to remove it
 Proof : 
 
![image](https://user-images.githubusercontent.com/1208311/109883483-c736a180-7c7b-11eb-8cc3-e0d933274d06.png)

![image](https://user-images.githubusercontent.com/1208311/109883537-dae20800-7c7b-11eb-9f56-31f7320afce1.png)

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/mangoszero/server/136)
<!-- Reviewable:end -->

Also added utf8limitmethod for future use.

MadMaxMangos commented:
:x: [Build Mangos ZERO Server 22.01.34-rixmvvsh failed](https://ci.appveyor.com/project/MaNGOS/server-9fytl/builds/38051410) (commit https://github.com/mangoszero/server/commit/e8cf4f92a6 by @Elmsroth)

Elmsroth commented:
Shit.. playerbots....

Elmsroth committed:
Fix HandleGMTicketUpdateTextOpcode

When updating the ticket , the client adds a leading '\a' char ! so we need to remove it

Fix build
